app/src/main/java/pl/skipcode/basekotlinapp/feature/main/fragments/dashboard/DashboardContract.kt

```diff
@@ -7,14 +7,14 @@ interface DashboardContract {
     interface View : BaseContract.View{
         fun showMessage(message: Int)
         fun clickLogout()
+        fun clickUsername()
         fun updateUsernameUI(username: String)
```

Comment:
tą metodę bym nazwał setUsernameText(username: String), bo tylko to ona robi i ta nazwa bardziej pasi